### ISSUE TITLE
Minor typo in Batch_Normalization_Lesson notebook

### ISSUE THREAD

**Issue by sbbogdan** [udacity/dlnd-issue-reports#477]
Issue: "It appears to hande the higher learning rate well, with both networks achieving high accuracy."

Resolution: "It appears to handle the higher learning rate well, with both networks achieving high accuracy."

'hande' --> should be --> 'handle'

**Comment by lcrucks** [comment]
Thanks for reporting!  It should be fixed now.
